dobi.yaml

```diff
@@ -406,6 +439,31 @@ job=run-grapl-analyzerlib-integration-tests:
     - grapl-analyzerlib-build
     - integration-env
 
+job=build-analyzer-deployer:
```

Comment:
i _would_ say that i don't love the naming scheme on a lot of these - like, `build-analyzer-deployer` versus `analyzer-deployer-build` - i almost wonder if we should name the images like `analyzer-deployer-build-img` or something

Reply:
Yeah adding `-img` and `-job` be better. I was thinking "nouns for images, verbs for jobs" initially but that's not 100% obvious.